packages/@aws-cdk/aws-eks/lib/kubectl-layer.ts

```diff
@@ -75,4 +77,8 @@ export class KubectlLayer extends Construct implements lambda.ILayerVersion {
   public addPermission(_id: string, _permission: lambda.LayerVersionPermission): void {
     return;
   }
+
+  public isChina(): boolean {
+    return this.stack.region.startsWith('cn-');
```

Comment:
This can only be done if region is not a token.
```suggestion
    const region = this.stack.region;
    return !Token.isUnresolved(region) && region.startsWith('cn-');
```